README.md

```diff
@@ -29,3 +29,25 @@ jobs:
 | target_branch | string | 'master' | Destination branch to push changes. |
 | force | boolean | false | Determines if force push is used. |
 | tags | boolean | false | Determines if `--tags` is used. |
+
+### GitHub Token
+
+#### Pull from public repository and push to current repository
+
+GitHub automatically creates a `GITHUB_TOKEN` secret to use in your workflow. 
+You can use the `GITHUB_TOKEN` to authenticate in a workflow run. 
+`github_token` input can passed in `${{ secrets.GITHUB_TOKEN }}`
```

Comment:
Is there any documentation about it? If yes should it be added as a reference as it is done below?